oneview_redfish_toolkit/api/scmb.py

```diff
@@ -110,19 +110,39 @@ def get_oneview_client():
         # multiple OneViews for events service
         ip=config.get_oneview_multiple_ips()[0],
         credentials=config.get_credentials(),
-        api_version=300
+        api_version=600
     )
     ov_client = OneViewClient(ov_config)
     ov_client.connection.login(config.get_credentials())
 
     return ov_client
 
 
-def get_cert():
+def _get_ov_ca_cert(ov_client):
+    URI = '/rest/certificates/ca'
+    resource_client = ResourceClient(ov_client.connection, URI)
+    cert = resource_client.get(URI + "?filter=certType:INTERNAL")
+    return cert
+
+
+def _get_ov_ca_cert_base64data(ov_client):
+    cert = _get_ov_ca_cert(ov_client)
+    returnCert = None
+    if isinstance(cert, dict):
+        if 'members' in cert.keys():
+            for certObj in cert.get('members'):
+                if certObj and certObj.get('certificateDetails') and \
+                        certObj.get('certificateDetails').get('base64Data'):
+                    returnCert = certObj.get('certificateDetails').\
+                        get('base64Data')
+                    break
+    return returnCert
+
+
+def get_scmb_certs():
     # Get CA
     ov_client = get_oneview_client()
-
-    cert = ov_client.certificate_authority.get()
+    cert = _get_ov_ca_cert_base64data(ov_client)
```

Comment:
Maybe we could check if **cert** is None before write it at line 151:
`f.write(cert)`